pulpito/templates/run.html

```diff
@@ -53,7 +53,9 @@ <h2>
                 <a href="/{{ run.name }}/detail"> See detail view?</a>
               </h2>
             </div>
-
+            <h2>
+              <a href="http://qa-proxy.ceph.com/teuthology/{{ run.name }}/scrape.log">Want to see scrape.log?</a>
```

Comment:
I would write it like this to keep similar to the other hyperlinks.
```suggestion
              <a href="http://qa-proxy.ceph.com/teuthology/{{ run.name }}/scrape.log">See scrape.log?</a>
```